framework/kibocommerce/api/endpoints/wishlist/add-item.ts

```diff
@@ -0,0 +1,124 @@
+import getCustomerWishlist from '../../operations/get-customer-wishlist'
+import getCustomerId from '../../utils/get-customer-id'
+import type { WishlistEndpoint } from '.'
+import { normalizeWishlistItem } from '@framework/lib/normalize'
```

Comment:
We should always do relative paths inside the providers folder, otherwise build will fail for other providers